src/tactic/ui/panel/table_layout_wdg.py

```diff
@@ -7095,6 +7095,25 @@ def handle_load_behaviors(self, table):
 }
 
 
+spt.table.find_group_name_by_element = function(src_el, level) {
+
+    var table = src_el.getParent("tbody");
```

Comment:
before you call this ... try spt.table.set_table()
then try spt.table.get_table()